apollo-router/src/plugins/telemetry/formatters/json.rs

```diff
@@ -124,39 +124,41 @@ where
             let custom_attributes = ext.get::<LogAttributes>().map(|attrs| attrs.attributes());
             if let Some(custom_attributes) = custom_attributes {
                 #[cfg(test)]
-                let custom_attributes: BTreeMap<
-                    &opentelemetry::Key,
-                    &opentelemetry::Value,
-                > = custom_attributes.iter().collect();
-                for (key, value) in custom_attributes.iter().filter(|(k, _)| {
-                    let key_name = k.as_str();
+                let custom_attributes: Vec<&opentelemetry::KeyValue> = {
+                    let mut my_custom_attributes: Vec<&opentelemetry::KeyValue> =
+                        custom_attributes.iter().collect();
+                    my_custom_attributes.sort_by_key(|kv| &kv.key);
+                    my_custom_attributes
+                };
```

Comment:
I think this part is useless now as custom_attributes are already a vec right ?

Reply:
It's required to sort or the insta tests won't pass. This code is all #[cfg(test)]

Reply:
good idea to mark it cfg(test), it won't have any impact otherwise